consensus/polybft/validator_set.go

```diff
@@ -425,6 +463,30 @@ func (v *validatorSet) Len() int {
 	return len(v.validators)
 }
 
+// CalculateQuorum calculates max faulty voting power and quorum size for given voting power map
+func (v *validatorSet) calculateQuorum() error {
+	totalVotingPower := uint64(0)
+	for _, v := range v.votingPowerMap {
+		totalVotingPower += v
+	}
+
+	if totalVotingPower == 0 {
+		return ErrInvalidTotalVotingPower
+	}
+
+	maxFaultyVotingPower := (totalVotingPower - 1) / 3
+	v.quorumSize = 2*maxFaultyVotingPower + 1
```

Comment:
We should change this to 2/3 supermajority:
`2/3*totalVotingPower`

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/904/commits/f3573424174f3f8cdcf6c2ad31ac9dbfebb0ed44